data/members.json

```diff
@@ -151,5 +151,10 @@
     "name": "Andrew James",
     "feed": "https://ajames.dev/rss",
     "url": "https://ajames.dev"
-  }
+  },
+  {
+    "name": "Tori Hall",
+    "feed": null,
+    "url:" "https://toriii.neocities.org/"
+  },
```

Comment:
Hi Toriii! You'll need to remove this comma as it will break the JSON file 🥲

Reply:
Hi, thanks for pointing that out! I forgot that JSON doesn't allow for trailing commas, that should be resolved now :)